### DISCUSSION TITLE
ibax wallet recovery method definitive solution

### DISCUSSION THREAD

**Discussion by yasin001yk** [IBAX-io/go-ibax#56] [Ideas] [answered]
friends, those who have problems with the wallet, if they follow this way, everyone can save their wallet, first of all, you need to have the rescue words given when setting up the wallet, which is very simple. WE ENTER THE PASSWORD, it is so simple, you can re-enter your password and log in. I applied the method of recovering your wallet with recovery words without a private key.

**Comment by HalloKyle** [comment, marked as answer]
You can also check this link for a solution： https://github.com/IBAX-io/go-ibax/discussions/62

**Comment by hemresenol** [comment]
just that simple.👍
